internal/gardener/shoot/extender/tolerations_test.go

```diff
@@ -0,0 +1,41 @@
+package extender
+
+import (
+	gardener "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"testing"
+
+	imv1 "github.com/kyma-project/infrastructure-manager/api/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestTolerationsExtender(t *testing.T) {
+	t.Run("Extend Tolerations", func(t *testing.T) {
```

Comment:
I think we should have negative test case here - this key should not be added for any other region than "me-central2"